cloudbuild.yaml

```diff
@@ -147,4 +147,4 @@ secrets:
   secretEnv:
     SENTRY_AUTH_TOKEN: CiQAVYLHlum+JbxvLUXADsLiiIwXjocXFGg5SAsxKyvAkDRjEaESaABnxmN4TReBWF0cY+XPS8xytlGSUhClZewwH7ttpBFKC65+Y0a6W8XIsTjyQMyVzWiLxSOef46bmI1Bo8JIbh/ErIRWm3rPBNFs843z52oF8M93+miMUlt8UOWmIgyq1U/fjThhq0UI
 
-timeout: 1200s
+timeout: 1800s
```

Comment:
Do you know why it is necessary to increase the timeout?

Reply:
No I don't know why. The two commits that timed out were 4c911de and f2f14ff. For each of the longer builds, I looked at the timing of each step and it looks like 3 of the steps took about twice as long as to complete they normally do: build-image, build-test-image, and run-tests. I have no idea why the changes I made would cause this. Interestingly, the build time went back to normal after your last merge into develop (97e690c), so maybe it was a temporary thing? We could revert this back to 1200s if future builds continue to stay under 20 minutes.